devnet-sdk/system/periphery/go-ethereum/fees.go

```diff
@@ -83,3 +118,17 @@ type EIP1159FeeEthClient interface {
 	BlockByNumber(ctx context.Context, number *big.Int) (*types.Block, error)
 	SuggestGasTipCap(ctx context.Context) (*big.Int, error)
 }
+
+// multiplyBigInt is a little helper for a messy big.Int x float64 multiplication
+//
+// It rounds the result away from zero since that's the lower risk behavior for fee estimation
+func multiplyBigInt(b *big.Int, m float64) *big.Int {
```

Comment:
nit: some unit tests for this would be good

Reply:
Added!